$NetBSD: patch-MANIFEST,v 1.1 2019/09/09 12:21:44 tm Exp $

Add missing MANIFEST file so that Munin::Common becomes manageable
by the Perl module framework.

--- /dev/null	2011-01-09 11:43:05.000000000 +0000
+++ common/MANIFEST
@@ -0,0 +1,22 @@
+Build.PL
+lib/Munin/Common/Config.pm
+lib/Munin/Common/Defaults.pm
+lib/Munin/Common/Timeout.pm
+lib/Munin/Common/TLS.pm
+lib/Munin/Common/TLSClient.pm
+lib/Munin/Common/TLSServer.pm
+MANIFEST			This list of files
+t/critic.t
+t/munin_common_config.t
+t/munin_common_defaults.t
+t/munin_common_timeout.t
+t/munin_common_tls.t
+t/pod-coverage.t
+t/pod.t
+t/tls/CA/ca_cert.pem
+t/tls/master_cert.pem
+t/tls/master_key.pem
+t/tls/node_cert.pem
+t/tls/node_key.pem
+t/tls/openssl.cnf
+t/tls/README
